pyproject.toml

```diff
@@ -30,6 +30,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 space_packet_parser = ">=4.0.2,<5"
+bitstring = '<=4.0.1'
```

Comment:
I think `bitstring` works with any version of your code, it is the combination of `bitstring + space_packet_parser` that isn't working. I'm not sure how to explicitly call that out here though... Just indicating that I don't think it is ideal to pin `bitstring` back when it is really `space_packet_parser` that is causing the failures.

Reply:
Indeed, I don't know what else to do though.  If you do a regular `poetry install `without specifying bitstring version here, the code doesn't work. Gavin is working on a fix, we could just wait for that before merging.

Reply:
Gavin already pushed a new version to pypi!